modules/administration-guide/partials/proc_configuring-gitlab-oauth2.adoc

```diff
@@ -0,0 +1,49 @@
+[id="configuring-gitlab-oauth2_{context}"]
+= Configuring GitLab OAuth2
+
+//[role="_abstract"]
+
+.Prerequisites
+
+* The GitLab server is running and available for {prod-short}.
+
+.Procedure
+
+. Follow GitLab's instructions for link:https://docs.gitlab.com/ee/integration/oauth_provider.html[configuring GitLab as an OAuth 2.0 authentication identity provider] to create an OAuth2 link:https://docs.gitlab.com/ee/integration/oauth_provider.html#authorized-applications[Authorized application] in Gitlab.
+
+. Enter {prod-short} as the application `Name`.
```

Comment:
@xbaran4, in your original, this sentence began with "Enter Che as the application `Name`. I replaced `Che` with `{prod-short}`: is this still correct? Can the downstream docs say "Enter Codeready Workspaces as the application `Name`? Or should I stick to `Che` here?

Reply:
I checked if Gitlab allows for multi-word application name and it does. So your change is correct.